app/helpers/application_helper.rb

```diff
@@ -23,4 +23,5 @@ def display_profile_warning?
   def display_search_ui?
     current_user && current_user.client_model && !client_disabled?
   end
+
```

Comment:
looks like you added newlines after blocks in a few files - I generally like newlines before/after multi-line blocks *except* when the end the block is directly nested inside another block (eg: two `end`s next to each other) what do you think?